dependencies required for mozjs

since recent commits, the build will fail with C++ errors. Either in Win10, or Ubuntu 16.04, as far as I know.

I add `apt install clang` and fix the Ubuntu. But I don't know how to fix Windows.

may be related to msvc version? this is message I got on local machines for reference.


```
cargo:rustc-link-search=native=C:\github\servo\target\debug\build\mozjs_sys-e622a19cc3f27e25\out
TARGET = Some("x86_64-pc-windows-msvc")
Generting bindings ["./src/jsglue.hpp", "--rust-target", "1.19", "--rustified-enum", ".*", "--no-derive-default", "--enable-cxx-namespaces", "--generate", "function,types,vars", "--ignore-methods", "--", "-I", "C:\\github\\servo\\target\\debug\\build\\mozjs_sys-e622a19cc3f27e25\\out\\dist/include", "-x", "c++", "-fms-compatibility", "-DRUST_BINDGEN", "-DSTATIC_JS_API", "-std=c++14", "-DWIN32"].

--- stderr
C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\VC\Tools\MSVC\14.14.26428\include\yvals.h:297:5: error: STL1000: Unexpected compiler version, expected Clang 6 or newer.
```

--- a/python/servo/packages.py
+++ b/python/servo/packages.py
@@ -4,7 +4,7 @@
 
 WINDOWS_MSVC = {
     "cmake": "3.7.2",
-    "llvm": "4.0.0",
+    "llvm": "6.0.0",
     "moztools": "0.0.1-5",
     "ninja": "1.7.1",
     "openssl": "1.1.0e-vs2015",

